bin/www

```diff
@@ -0,0 +1,7 @@
+var app = require('../app');
+
+
+const port = 3000;
+app.listen(port, () => {
+    console.log(`Listening to port ${port}`)
+})    
```

Comment:
@Wajahat-Khan add an empty line at end of your file. Github asks for an empty line at end.